crates/vault-registry/src/lib.rs

```diff
@@ -1544,6 +1565,20 @@ impl<T: Config> Pallet<T> {
         collateral.try_into().map_err(|_| Error::<T>::TryIntoIntError.into())
     }
 
+    pub fn get_max_nomination_ratio() -> Result<UnsignedFixedPoint<T>, DispatchError> {
+        let secure_collateral_threshold = Self::secure_collateral_threshold();
+        let premium_redeem_threshold = Self::premium_redeem_threshold();
+        Ok(secure_collateral_threshold
```

Comment:
What is the logic behind this? I'm guessing it's explained in the spec, but a comment here would be helpful I think